src/components/SimpleDialog.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <modal-template v-if="dialog.isRevealed.value">
+    <template #message>
+      <paragraph>
+        {{ message }}
+      </paragraph>
+    </template>
+
+    <template #menu>
+      <li>
+        <button
+          modelValue
+          @click="dialog.cancel"
+          class="utrecht-button utrecht-button--secondary-action"
+        >
+          {{ cancelMessage }}
+        </button>
+      </li>
+      <li>
+        <button
+          modelValue
+          @click="dialog.confirm"
+          class="utrecht-button utrecht-button--action"
+        >
+          {{ confirmMessage }}
+        </button>
+      </li>
+    </template>
+  </modal-template>
+</template>
+
+<script setup lang="ts">
+import Paragraph from "@/nl-design-system/components/Paragraph.vue";
+import type { ComputedRef, PropType } from "vue";
+import ModalTemplate from "./ModalTemplate.vue";
+defineProps({
+  dialog: {
+    type: Object as PropType<{
+      isRevealed: ComputedRef<boolean>;
+      confirm: () => void;
+      cancel: () => void;
+    }>,
+    required: true,
+  },
```

Comment:
The dialog that we provide actually comes from a library called `@vueuse`. However, it doesn't export the type returned by the method `useConfirmDialog`. I will put the type definition in an actual type with a descriptive name, I think this will help.